de.gonicus.gonnect.yml

```diff
@@ -0,0 +1,96 @@
+id: de.gonicus.gonnect
```

Comment:
how do I know you control this https://www.gonicus.de/ ? The domain in github is not verified and I can't find a mention of this app on the website

Reply:
I thought that you hand over a token, and I place it in the well knowns. If you require the github organization to be verified, I'll do that. Give me some minutes when I'm in the office...

Reply:
github organization is verified now.

Reply:
that's after the submission is accepted, now I need to know that you aren't using a random domain that's not yours (people tried that)